docs/ldml/tr35-keyboards.md

```diff
@@ -982,6 +988,8 @@ _Attribute:_ `base`
 _Attribute:_ `path` (required)
 
 > If `base` is `cldr`, then the `path` must start with a CLDR version (such as `techpreview`) representing the CLDR version to pull imports from. The imports are located in the `keyboard/import` subdirectory of the CLDR source repository.
+> Implementations are not required to have any or all all CLDR versions available to them.
```

Comment:
```suggestion
> Implementations are not required to have any or all CLDR versions available to them.
```

But given the statement above "other than required standard import data from the CLDR repository", this seems disingenuous?

Reply:
The implied import data is required, so perhaps this should just be 'all'.